### ISSUE TITLE
Design For Tutorial

### ISSUE THREAD

**Issue by meilynbear** [meilynbear/beginnersmakelistpractice#1]
Not an issue, but more of a discussion on how this tutorial could be improved!

## Add images to show each step
This might be nice to do because most people are visual learners.
## Make a longer tutorial
Throughly go through each step rather than skimming over everything

#### Feel free to discuss and add on to whatever you think would make this tutorial easier to navigate!

**Comment by softacoder** [comment]
I think to also use instruction videos to complement the images would be helpful.

**Comment by xentigen** [comment]
> hey is this avilable to contribute

Yeah I Think so, as you can see on its [README.md](https://github.com/meilynbear/beginnersmakelistpractice/blob/main/README.md):

<img width="934" height="806" alt="Image" src="https://github.com/user-attachments/assets/9ed13e49-fe41-44f8-8b17-3175bd41a627" />

There fore you may also want to read its license and code of conduct

<img width="1619" height="850" alt="Image" src="https://github.com/user-attachments/assets/a9321732-3713-4b22-a147-80a9fd6b580e" />

**Comment by xentigen** [comment]
> Not an issue, but more of a discussion on how this tutorial could be improved!
> 
> ## Add images to show each step
> This might be nice to do because most people are visual learners.
> 
> ## Make a longer tutorial
> Throughly go through each step rather than skimming over everything
> 
> #### Feel free to discuss and add on to whatever you think would make this tutorial easier to navigate!


how about a translation, but uh i dont know this will be useful or not. Also Im not an english native speaker actually
